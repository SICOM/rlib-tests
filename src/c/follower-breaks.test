COMMAND=./follower-breaks
FORMATS="txt"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
