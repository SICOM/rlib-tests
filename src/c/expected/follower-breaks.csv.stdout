"First Name          ","Last Name           ","Color               "
"1                    Bob                 ","Doan                ","blue                 WRD   MATCH    MATCH    MATCH    MATCH   "
"1                    Terry               ","Doan                ","blue                 TD    MATCH    MATCH    MATCH    MATCH   "
"1                    Eric                ","Buruschkin          ","green                ERB   MATCH    MATCH    MATCH    MATCH   "
"2                    Mike                ","Roth                ","yellow                     NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
"2                    Bob                 ","Kratz               ","pink                       NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
"2                    Steve               ","Tilden              ","purple                     NO_MATCH NO_MATCH NO_MATCH NO_MATCH"
